### PR TITLE
Fix typo in executor

### PR DESCRIPTION
**Pull request by pablogs9**
Hello @ralph-lange and @JanStaschulat, @jamoralp  found this: 

Our non-german readers are sad :(

image

### CODE REVIEW

File: _docs/concepts/client_library/real-time_executor/index.md [hunk 1]
```diff
@@ -181,12 +181,12 @@ An Alternative would be to evalute the IMU sample and the laser scan by synchron
 Figure 6: Synchronization of multiple input data with a trigger.
 
 In ROS 2, this is currently not possible to be modeled because of the lack of a trigger concept in the ROS2 Executor. Message filters could be used to synchronize input data based on the timestamp in the header, but this is only available in rclcpp (and not in rcl). Further more, it would be more efficient to have such a trigger concept directly in the Executor.
-<!--
-TODO
+
+<!--TODO
 - Bilder erweitern mit drei boxen: request IMU, process laser, fusion
```
Comment (ralph-lange) [inline _docs/concepts/client_library/real-time_executor/index.md:186]:
```suggestion
```

File: _docs/concepts/client_library/real-time_executor/index.md [hunk 2]
```diff
@@ -181,12 +181,12 @@ An Alternative would be to evalute the IMU sample and the laser scan by synchron
...
   dann wird klarer was mit den Daten wird
```
Comment (ralph-lange) [inline _docs/concepts/client_library/real-time_executor/index.md:187]:
```suggestion
```

File: _docs/concepts/client_library/real-time_executor/index.md [hunk 3]
```diff
@@ -181,12 +181,10 @@ An Alternative would be to evalute the IMU sample and the laser scan by synchron
...
 
 In ROS 2, this is currently not possible to be modeled because of the lack of a trigger concept in the ROS2 Executor. Message filters could be used to synchronize input data based on the timestamp in the header, but this is only available in rclcpp (and not in rcl). Further more, it would be more efficient to have such a trigger concept directly in the Executor.
-<!--
-TODO
-- Bilder erweitern mit drei boxen: request IMU, process laser, fusion
-  dann wird klarer was mit den Daten wird
-- Aus dem Bild die Beschreibung löschen
--->
+
+<!--TODO
```
Comment (ralph-lange) [inline _docs/concepts/client_library/real-time_executor/index.md:185]:
```suggestion
```

File: _docs/concepts/client_library/real-time_executor/index.md [hunk 4]
```diff
@@ -181,12 +181,10 @@ An Alternative would be to evalute the IMU sample and the laser scan by synchron
...
+- Aus dem Bild die Beschreibung löschen -->
```
Comment (ralph-lange) [inline _docs/concepts/client_library/real-time_executor/index.md:186]:
```suggestion
```

**Comment by ralph-lange** [comment]
The list can be removed completely. (Just did it.)

**Review by ralph-lange — APPROVED** [review]

**Comment by ralph-lange** [comment]
@pablogs9 and @jamoralp, thank you for pointing to this issue!

**Review by JanStaschulat — COMMENTED** [review]
Upps. Sorry.

**Pull request merged**